src/components/FullDetailView/HeaderSection.js

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) Microsoft Corporation and others. Licensed under the MIT license.
```

Comment:
This and the other new files are not inherently part of the full details view. they are used there but they may be used elsewhere.Is there anything stopping them from being moved up a level and added to top level index.js?

Reply:
If you think that HeaderSection and the other render components could be reused somewhere else, we could definitely move them as reusable components.

Reply:
I'd flip this around and say "if you think they **can't** be used anywhere else, leave them here. Otherwise, move them to be reusable"